.github/workflows/issue-triage.yml

```diff
@@ -94,6 +96,10 @@ jobs:
               This comment acknowledges we believe this may be a bug and there’s enough information to investigate it.
               However, we can’t promise any sort of timeline for resolution. We prioritize issues based on severity, breadth of impact, and alignment with our roadmap. If you’d like to help move it more quickly, you can continue to investigate it more deeply and/or you can open a pull request that fixes the cause.
             `})
+      - name: Remove "Needs Review" label
+        uses: actions-ecosystem/action-remove-labels@v1
+        with:
+          labels: 'needs review'
```

Comment:
we don't need 3rd party action to remove a label. using the github.rest api is good enough. e.g.
https://github.com/expo/expo/blob/2740dee1a618e0e5c5ffd66a0cbe3c873b932ed9/.github/workflows/pr-labeler.yml#L63-L74

Reply:
Sure thing - updated! ✔️